### PR TITLE
Release notes for beta-20160519

### PR DESCRIPTION
**Pull request by bdarnell**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/303)

### CODE REVIEW

**Comment by jseldess** [comment]
:lgtm:, I'll do minor copy edits and add links and doc updates once this is merged.

<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary>
> #### Release notes for beta\-20160519
</details>


---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/303#-:-KI8O8irmGx0RT2wUMj4:906038724)*

**Comment by mberhault** [comment]
LGTM

File: beta-20160519.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  [#6633](https://github.com/cockroachdb/cockroach/pull/6633)
+* `CHECK` constraints can now be specified at the table level.
+  [#6625](https://github.com/cockroachdb/cockroach/pull/6625)
+* Binary encoding is now supported in the network protocol for `BOOL`,
+  `FLOAT`, `DECIMAL`, and `STRING` types, improving compatiblity with
+  some PostgreSQL drivers.
+  [#6661](https://github.com/cockroachdb/cockroach/pull/6661)
+* A new command `cockroach halt-cluster` has been added to the
+  command-line interface; this will be used in the upgrade process for
+  a future beta releasea.
```
Comment (RaduBerinde) [inline beta-20160519.md]: release

Comment (jseldess) [inline beta-20160519.md]: Thanks, Radu. I'll correct this in my copyedit PR.

**Comment by nvanbenschoten** [comment]
Could you double check that https://github.com/cockroachdb/cockroach/pull/6752 went in after this change. Is there are good way to verify stuff like this on our own in the future (i.e. check the SHA of the release)?

**Comment by mberhault** [comment]
This particular change did not make it in.
The beta is using 5430fae20fd2418260ed7fb0299b1aa1ae3af4bf

In general, you can see which sha is being used by looking for the latest `beta-YYYYMMDD` issue. #6770 for this week. It mentions the candidate sha being tested.

**Comment by nvanbenschoten** [comment]
Perfect, thanks Marc.

**Pull request merged**